tools/Makefile

```diff
@@ -80,7 +80,8 @@ all_unittest_code = $(fruit_code) $(unittest_code) $(unittest_code_gen)
 fruit_driver = $(UNITTESTDIR)/fruit_driver.exe
 
 $(UNITTESTDIR)/fruit_basket_gen.f90 : $(unittest_code)
-	@cd $(UNITTESTDIR) ; ruby fruit_generator.rb
+	cp tools/fruit_generator.rb $(UNITTESTDIR)
+	@cd $(UNITTESTDIR); sed -i "4s,.*,load \"$(FRUITDIR)/rake_base.rb\"," fruit_generator.rb; ruby fruit_generator.rb
```

Comment:
What do you think? I have used the commas in sed for consistency with other sed commands.

Reply:
Looks decent. I think the `cp` might want to be `@cp`, and the existing `fruit_generator.rb` needs to be deleted from the repo. Could you also add a comment to document what this does and why, as it's probably not obvious?